Sources/Sentry/SentryEnvelope.m

```diff
@@ -129,6 +129,24 @@ - (instancetype)initWithUserFeedback:(SentryUserFeedback *)userFeedback
                            data:json];
 }
 
+- (instancetype)initWithFeedback:(SentryFeedback *)feedback
```

Comment:
`l`: This method is almost identical to `initWithUserFeedback`. Maybe it makes sense to extract a common method for them.

Reply:
Same as https://github.com/getsentry/sentry-cocoa/pull/4535#discussion_r1847524583, at this point I just want to keep things separate so it's easier to remove the other thing later.